OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,422 @@
+
+To mount a host directory as a data volume, execute the below commands as **root** user.
+```
+   $ mkdir -p /scratch/DockerVolume/WCSitesVolume/WCSites /scratch/DockerVolume/WCSitesVolume/WCSitesShared
+   $ chmod -R 777 /scratch/DockerVolume/WCSitesVolume
+```
+All container operations are performed as 'oracle' user. Therefore, the ‘chmod’ command must be run to grant permission to all the non-root users.
+
+### C. Setting up an Oracle Database Docker container
+To set up an Oracle Database Docker container, you must first update the environment file which is passed as a parameter in the command that starts the database container.  
+
+##### 1. Update the Environment File
+Update the environment **db.env.list** file, to define the parameters, which is located at `../docker-images-master/OracleWebCenterSites/dockerfiles/db.env.list`.
+
+db.env.list file details:
+```
+   DB_SID=ORCLCDB
+   DB_PDB=ORCLPDB1
+   DB_DOMAIN=localdomain 
+```
+##### 2. Start the Database Container
+To start a database container run these commands:
+```
+   $ cd ../docker-images-master/OracleWebCenterSites/dockerfiles
+        
+   $ docker run -d --name <container_name> --network=<network_name> -p <database_listener_port>:1521 -p <enterprise_manager_port>:5500 --env-file <environment_file> <repo_name:tag_name>
+```
+Sample command:
+```
+   $ docker run -d --name WCSites12212Database --network=WCSitesNet -p 1521:1521 -p 5500:5500 --env-file ./db.env.list database/enterprise:12.2.0.1
+```
+Database start up command explained:
+```
+   | --name                         | container_name         | Database name; set to ‘WCSites12212Database’                                         |
+   | --network                      | network_name           | User-defined network to connect to; use the one created earlier ‘WCSitesNet’         |
+   | -p                             | database_listener_port | Database listener port; set to ‘1521’. Maps the container port to Docker host port.  |
+   | -p                             | enterprise_manager_port| Enterprise Manager port; set to ‘5500’. Maps the container port to Docker host port. |
+   | database/enterprise:12.2.0.1-ee| repo_name:tag_name     | Repository name, Tag name of the image.                                              |
+```
+Running the above command creates a Container Database (CDB) with one Pluggable Database (PDB).
+
+This is the Database connection string:
+```
+   DB_CONNECTSTRING=<hostname/containername>:1521/ORCLPDB1.localdomain
+```   
+   **Note**: Container name can be given only if the container is located on the same host machine. Ensure SERVICE_NAME is a valid PDB service name in your database as given in the $ORACLE_HOME/admin/ORCLCDB/tnsnames.ora file.
+   
+Additional information on [running Oracle Database image](https://container-registry.oracle.com)
```

Comment:
You're linking to the home page of the Container Registry here which doesn't provide any information on running an Oracle Database _container_.